.github/workflows/dispatch-bazaar.yml

```diff
@@ -19,7 +19,9 @@ jobs:
     run: |
       echo "::set-output name=category::${{ github.event.discussion.category.name }}"
       echo "::set-output name=title::${{ github.event.discussion.title }}"
-      echo "::set-output name=body::${{ github.event.discussion.body }}"
+      
+      echo "${{ github.event.discussion.body }}" > body
```

Comment:
Anyway, here we are very weak against `discussion.body` containing special chars like `"` and/or `'`.

Reply:
Indeed, but we can try this out with a dispatch

Reply:
Judging from this [log](https://github.com/robotology/community/runs/2974052048?check_suite_focus=true), no way...

Maybe @AlexAntn knows how to deal with that.

Reply:
Yep no good. @ilaria-carlini will also be able to give some hints on the parsing special chars.